cmd/kube-proxy/app/server.go

```diff
@@ -1046,7 +1046,7 @@ func getNodeIPs(logger klog.Logger, client clientset.Interface, name string) []n
 	}
 
 	err := wait.ExponentialBackoff(backoff, func() (bool, error) {
-		node, err := client.CoreV1().Nodes().Get(context.TODO(), name, metav1.GetOptions{})
+		node, err := client.CoreV1().Nodes().Get(context.TODO(), name, metav1.GetOptions{ResourceVersion: "0"})
```

Comment:
add a comment please explaining this, the magic resource version 0 is not well known and people reading the code as is may not understand it

I think is ok,  but prefer if @wojtek-t ack

Reply:
Maybe an overkill, but I am thinking to use helper function named [FromApiserverCache](https://github.com/kubernetes/kubernetes/blob/v1.29.1/pkg/kubelet/util/util.go#L29) as the function name is self-explanatory. However, the function is now in pkg/kubelet/util/, which is less suitable to be used outside of kubelet.

Reply:
comment is better, growing helper functions ends creating dependencies and worse code organization, also the risk the function changes ... 
```suggestion
                // Improve the efficiency by setting ResourceVersion "0", that means "serve the current version held by the API server cache",
		node, err := client.CoreV1().Nodes().Get(context.TODO(), name, metav1.GetOptions{ResourceVersion: "0"})
```

Reply:
Make sense, thanks!